docs/tools/terraform/getting-started.rst

```diff
@@ -0,0 +1,98 @@
+Set up your first Terraform project
+===================================
+
+This example shows the setup for a sample Terraform project and some useful commands to stand up (and destroy) your Aiven data infrastructure.
+
+Requirements 
+''''''''''''
+- `Download and install Terraform <https://www.terraform.io/downloads.html>`_
+- `Sign up <https://console.aiven.io/signup?utm_source=github&utm_medium=organic&utm_campaign=devportal&utm_content=repo>`_ for Aiven if you haven't already
+- `Generate an authentication token <https://developer.aiven.io/docs/platform/howto/create_authentication_token.html>`_ on Aiven's console or CLI
+
+
+1. ``provider.tf`` file:
+
+.. code:: bash
+
+   terraform {
+      required_providers {
+         aiven = {
+            source  = "aiven/aiven"
+            version = ">= 2.6.0, < 3.0.0"
+         }
+      }
+   }
+
+   provider "aiven" {
+      api_token = var.aiven_api_token
+   }
+
+
+Consider this code block similar to declaring a dependency; the *Aiven Terraform Provider* in this case. Within the *required_providers* block, you mention the source of the provider and specify a certain version. The versions here are for example purpose and will change in the future.
+Following `Aiven Terraform Provider doc <https://registry.terraform.io/providers/aiven/aiven/latest/docs>`_, *api_token* is the only parameter for the provider configuration.
+
+Make sure the owner of the API Authentication Token has admin permissions in Aiven.
+
+2. ``variables.tf`` file:
+
+.. code:: bash
+
+   variable "aiven_api_token" {
+      description = "Aiven console API token"
+      type = string
+   }
+
+   variable "project_name" {
+      description = "Aiven console project name"
+      type        = string
+   }
+
+Avoid including sensitive values in configuration files that are under source control. Rather than using sensitive information within this *variables.tf* file, you can use a ``*.tfvars`` file so that Terraform receives the values during runtime.
+
+3. ``var-values.tfvars`` file:
+
+.. code:: bash
+
+   aiven_api_token = "<YOUR-AIVEN-AUTHENTICATION-TOKEN-GOES-HERE>"
+   project_name = "<YOUR-AIVEN-CONSOLE-PROJECT-NAME-GOES-HERE>"
+
+This is where you put the actual values for Aiven API token and Aiven console project name. This file is passed to Terraform using the ``-var-file=`` flag.
+
+4. ``services.tf`` file:
```

Comment:
This is a bit open-ended for actually getting started for a new person. Let's explain that you can use a number of `*.tf` files to describe your services, and perhaps provide the most simple thing (one Redis service?) in our example.